src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -25,13 +29,18 @@
     private final File file = new File(storagePath);
     private final String filePath = file.getAbsolutePath();
     private WorkoutListModel workoutListModel = new WorkoutListModel();
+    private static final Logger LOGGER = Logger.getLogger(Storage.class.getName());
 
     public Storage() throws GetJackDException {
+        setupLogger(LOGGER);
         try {
+            assert file != null;
             file.getParentFile().mkdirs();
             file.createNewFile();
+            LOGGER.info("Generated Data Folder and storage JSON");
         } catch (IOException e) {
-            throw new GetJackDException("☹ OOPS!!! Data file can't be found.");
+            LOGGER.log(Level.SEVERE, "Data file can't be created. Exception: ", e);
```

Comment:
Here we could change the level to Warn maybe? cause the console handler is setup to log into the terminal when the level is Severe and we are already printing the exception error message to the user so having 2 error messages could be confusing.
Alternatively, we could not add the console handler to the logger in `setupLogger()` so there's no logs being printed to the user in any case